grid/src/js/views/alert.js

```diff
@@ -0,0 +1,19 @@
+import $ from "jquery";
+
+export default class alertView {
+  constructor() {
+    this.$alert = $('#alert');
+  }
+
+  show(e) {
+    this.$alert.html(this.template(e));
+    this.$alert.fadeIn();
+
+    setTimeout(() => this.$alert.fadeOut(), 5000);
+  }
+
+  template(error) {
+    const template = `<strong> ${error} </strong><br> Pls try again`;
```

Comment:
Let's use classes for text highlighting or bolding and so on.
Pls doesn't look user friendly for me